test/unit/cute/CMakeLists.txt

```diff
@@ -105,6 +109,9 @@ else()
     cutlass_test_unit_cute_layout
     cutlass_test_unit_cute_core
     cutlass_test_unit_cute_msvc_compilation
+    if(SYCL_INTEL_TARGET)
```

Comment:
We can't add and `if(SYCL_INTEL_TARGET)` here. CMake will fail. Instead you can just wrap the `cutlass_test_unit_add_executable` in `test/unit/cute/intel_xe/CMakeLists.txt` with an  `if(SYCL_INTEL_TARGET)`

```cmake
if(SYCL_INTEL_TARGET)
cutlass_test_unit_add_executable(
  cutlass_test_unit_cute_intel_xe
  copy_1d.cpp
  copy_subgroup_block.cpp
  copy_block.cpp
  copy_scatter.cpp
  mma.cpp
  gemm_partition_src_dst.cpp
  gemm_partition_fragment_abc.cpp
  gemm_tiled_copy_abc.cpp
  gemm_row_col.cpp
  gemm_col_row.cpp
  gemm_col_col.cpp
)
endif()
```

Reply:
You can run the test with `ninja test_unit` or `make test_unit`